tests/rados/test_noscrub_nodeepscrub_check.py

```diff
@@ -0,0 +1,218 @@
+"""
+The file contains the script that verify the customer scenario of scrub and deep scrub functionality
```

Comment:
please add the doc text from bug. This will help with understanding the test workflow.

```

Previously, in some cases, using the `noscrub` or `nodeep-scrub` flags resulted in incorrect handling of these flags. As a result, placement groups (PGs) were not scrubbed until the primary OSD was restarted. In addition, until the OSD was restarted, the number of concurrent scrubs that could be performed by any of the OSDs that are storing the PG was reduced. These OSD limitations were on both the primary OSDs and OSDs serving as replicas.  If the maximum configuration was 1, which was the default, the affected OSDs would not be able to participate in any new scrub actions, for any PG.

With this fix, the 'noscrub' flag now works as expected.
```

Reply:
Added the doc text.